src/Tests/Microsoft.Diagnostics.Monitoring.Tool.UnitTests/DiagnosticPortTestsHostHelper.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Diagnostics.Monitoring.WebApi;
+using Microsoft.Diagnostics.Tools.Monitor;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Options;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Xunit.Abstractions;
+
+namespace Microsoft.Diagnostics.Monitoring.Tool.UnitTests
+{
+    internal static class DiagnosticPortTestsHostHelper
```

Comment:
This is closely modeled after `TestHostHelper`; I originally planned to integrate with that, but decided it carried a lot of things that aren't relevant for this use-case and made a separate version for this set of tests.

Reply:
It's okay that `TestHostHelper` adds other services and configuration so long as it doesn't impact the diagnostic port testing; I'd rather reuse the common class rather than copying it and slimming it down for each individual use case.